api/endpoints/mail.md

```diff
@@ -355,35 +355,10 @@ Collection of all Mail Items.
 
 + Response 200
     
-    [Mail Items Collection][]
-     
-## Check In a Mail Item [POST /mail/items/checkin]
-To Check In a Mail Item send JSON with the required attributes.
-  
-+ Request (application/json)
-
-				{
-				  "itemType": "OUTBOUND",
-				  "packageType": {
-					"id": 1
-				  },
-				  "center": {
-					 "id": 62
-					},
-				  "fromUser": {
-					"id": 13
-				  },
-				  "toUser": {
-					"id": 126
-				  },
-				  "trackingNumber": "351987984"
-				}
-
-+ Response 200  
-    [Mail][]   
+    [Mail Items Collection][]  
     
 ## Check In many Mail Items [POST /mail/items/checkin]
```

Comment:
Good idea to combine these since they're essentially the same call--you can go ahead and change this header to match your new "one or many" verbiage, so it's easily apparent in the sidebar and subhead as well.

![image](https://user-images.githubusercontent.com/15133199/50660521-a3755500-0f65-11e9-9dcc-865224e0bcc5.png)